code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -273,6 +273,9 @@
 				SEND_SIGNAL(src, COMSIG_ADD_MOOD_EVENT, "friendly_hug", /datum/mood_event/besthug, M)
 			else if (mood.sanity >= SANITY_DISTURBED)
 				SEND_SIGNAL(src, COMSIG_ADD_MOOD_EVENT, "friendly_hug", /datum/mood_event/betterhug, M)
+			
+			if(isethereal(src) && ismoth(M))
+				SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "friendly_hug", /datum/mood_event/lamphug, src)
```

Comment:
could you make the ``"friendly_hug"`` and/or ``"hug"`` into defined constants

Reply:
It's like this all over the codebase D: but yea needs to be done eventually